README.md

```diff
@@ -176,6 +176,25 @@ usbDetect.stopMonitoring();
 
 Make sure you call `usbDetect.startMonitoring()` before any calls to `usbDetect.find()`.
 
+# Known problems:
+
+### If you are using this dependency with electron.js is posible that you see an error similar to: *detection.node was compiled against a different Node.js version using NODE_MODULE_VERSION 72. This version of Node.js requires NODE_MODULE_VERSION 80. Please try re-compiling or re-installing*
```

Comment:
Let's put this in a code block so the heading isn't so long/big.

Here is how I would format things


```
### Use with Electron (mismatched `NODE_MODULE_VERSION`)

If you try to use `usb-detection` with [electron](https://www.electronjs.org/), you run into the following error. You can follow the steps below to resolve the problem.

\```
detection.node was compiled against a different Node.js version using NODE_MODULE_VERSION 72. This version of Node.js requires NODE_MODULE_VERSION 80. Please try re-compiling or re-installing
\```

 1. ...
 1. ...
```